columns/Column.js

```diff
@@ -41,11 +41,14 @@ export class Column {
      * @param {boolean} [c.resizable] - false to prevent user from drag-and-drop resizing.
      * @param {boolean} [c.movable] - false to prevent user from drag-and-drop re-ordering.
      * @param {Column~rendererFn} [c.renderer] - function to produce a formatted string for each cell.
-     *      Supports HTML as output. Passed both field value and entire row data object.
+     *      Supports HTML as output. Passed the field value as the first parameter, and an object
```

Comment:
I think you should remove the detailed type info from this text, and rely on the jsdoc definition of Column~rendererFn instead (which is now out of date).